addons/project_todo/wizard/mail_activity_todo_create.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="mail_activity_todo_create_popup" model="ir.ui.view">
+        <field name="name">mail.activity.todo.create.popup</field>
+        <field name="model">mail.activity.todo.create</field>
+        <field name="arch" type="xml">
+            <form js_class="todo_activity_wizard">
+                <group>
+                    <field name="summary" default_focus="1" placeholder="Reminder to..." required="1"/>
```

Comment:
And if the field is required (in the python model), no need to add the attribute here :neckbeard:

Reply:
Ok, I'll do as you say :+1: